include/ur_api.h

```diff
@@ -1647,7 +1649,7 @@ typedef enum ur_device_info_t {
 ///     - ::UR_RESULT_ERROR_INVALID_NULL_HANDLE
 ///         + `NULL == hDevice`
 ///     - ::UR_RESULT_ERROR_INVALID_ENUMERATION
-///         + `::UR_DEVICE_INFO_INTEROP_SEMAPHORE_EXPORT_SUPPORT_EXP < propName`
```

Comment:
Just checking you meant to delete this line?

Reply:
Seems like it needs to be the last enumerator, as it's saying any value after that will result in `UR_RESULT_ERROR_INVALID_ENUMERATION`. I was skeptical at first too, but the generated script diff check insisted.

Reply:
Yeah, this is the generator making the change. It's expected.

Reply:
It could probably be a bit more clever to do proper validation of the enum ranges.